rts/lra/lra-client/src/main/java/io/narayana/lra/client/NarayanaLRAClient.java

```diff
@@ -230,6 +226,27 @@ public void setCurrentLRA(URI coordinatorUri) {
         }
     }
 
+    public List<LRAData> getAllLRAs() {
+        Client client = null;
+        try {
+            client = ClientBuilder.newClient();
```

Comment:
thanks, in NarayanaLRAClient this is possible to cache.

I'll just note that I've found it difficult to cache the client in the LRARecords as they are persisted and recovered from ObjectStore this is kind of difficult. I would like to investigate more in the future.

Reply:
So it's not, probably just synchronization issues and the same client is used concurrently in different enlistments for instance. Worth investigating.

Reply:
My comment was
> "Can you move the Client construction code to a separate method so that we can cache it in the future (you don't need to cache it in this PR but we should be doing so in the future)"

ie I am not asking you to cache the client in this PR.

Reply:
Since my comments do not effect the functionality I am resolving this conversation.